package.json

```diff
@@ -22,31 +22,41 @@
     "lint-php": "composer run lint",
     "format-js": "eslint --fix assets/js",
     "lint": "npm run lint-css && npm run lint-js && npm run lint-php",
-    "format": "npm run format-js"
+    "format": "npm run format-js",
+    "cypress:open": "cypress open --browser chrome --config-file tests/cypress/cypress.config.js",
+    "cypress:run": "cypress run --browser chrome --config-file tests/cypress/cypress.config.js",
+    "env": "wp-env",
+    "env:start": "wp-env start",
+    "env:stop": "wp-env stop",
+    "postenv:start": "./tests/bin/initialize.sh"
   },
   "dependencies": {
+    "cypress-file-upload": "^5.0.8",
```

Comment:
We may remove this as we already have `.selectFile()` in Cypress and no longer needed this dependency. need to update `.attachFile()` to `.selectFile()`

More info: https://docs.cypress.io/guides/references/migration-guide#Migrating-from-cypress-file-upload-to-selectFile